storybookjs/storybook
# Addon-docs: Apply list styles over reset

Closes #10270 

## What I did

Force a default `list-style` for un.ordered lists with addon-docs.
Tailwind or Bulma, for example, used to remove default style so let's define it, to be sure we display lists if these CSS frameworks are globally included.

## How to test

- Is this testable with Jest or Chromatic screenshots? It should not change anything 🤞 
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No
